static/output.js

```diff
@@ -59,15 +59,19 @@ define(function (require) {
             this.add(obj.text, obj.tag ? obj.tag.line : obj.line);
         }, this);
 
-        this.add("Compiler exited with result code " + result.code);
```

Comment:
I'm not too sure about the removal of the result code. What was your motivation behind it? I do think it is useful even in the "success" case

Reply:
My reasoning was that the program will not execute unless the compilation return code is 0, so that is redundant information.

Reply:
But what will happen if no execution is requested? You now dont have what the compiler returned, right?